virttest/qemu_qtree.py

```diff
@@ -216,8 +217,11 @@ def generate_params(self):
             self.params['image_name'] = os.path.realpath(
                 self.block.get('backing_file'))
```

Comment:
Is it needed for getting `backing_file`, too?

Reply:
Thanks, have updated.
@XueqiangWei  could you help me to confirm and review? thanks.